crates/chain/src/lib.rs

```diff
@@ -59,6 +64,8 @@ impl Chain {
             Self::Sepolia => Duration::from_millis(12_000),
             Self::ArbitrumOne => Duration::from_millis(250),
             Self::Base => Duration::from_millis(2_000),
+            // dummy value as hardhat chain doesn't have a fixed block time
+            Self::Hardhat => Duration::from_millis(1_337),
```

Comment:
Do the tests pass if you put `panic!("no default values for <PARAMETER> available on hardhat")` for all the cases you currently return dummy values for hardhat?
I think hitting the default code paths would be an error, no?

Reply:
Adjusted. Should be fine that way as well. Let's see if tests pass